client/src/components/CategoryList.js

```diff
@@ -0,0 +1,110 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+import { ReactComponent as Bamboo } from "../assets/icons/bamboo.svg";
+import { ReactComponent as Travel } from "../assets/icons/travel.svg";
+import { ReactComponent as Bath } from "../assets/icons/bathroom.svg";
+import { ReactComponent as Gift } from "../assets/icons/gift.svg";
+import { ReactComponent as Household } from "../assets/icons/household.svg";
+import { ReactComponent as Ocean } from "../assets/icons/ocean.svg";
+import { ReactComponent as Reuse } from "../assets/icons/reuse.svg";
+import { ReactComponent as Tree } from "../assets/icons/tree.svg";
```

Comment:
nice :) never saw this kind of import for a svg! Ill mark that as awesome 🚀

Reply:
I was very happy when I found it! Makes styling of SVGs so much easier :)